docs/index.md

```diff
@@ -17,4 +17,5 @@ address to the issue ✅&nbsp;[ComPWA/gluex-nstar#1](https://github.com/ComPWA/g
 ```{toctree}
 reaction
 pgamma-state
+all-in-one
```

Comment:
Let's rename this to something like `ppi0eta`.

Reply:
At some stage, I would introduce a folder hierarchy. Something like `reaction/` (QRules stuff), `amplitude/` (symbolic stuff), `fit/`. Maybe this is a good place to start with that: `amplitude/ppi0eta.ipynb`.

Reply:
Let's wait with this until I review the notebook through Hypothesis